docs/2-attack-of-the-pipelines/3b-tekton.md

```diff
@@ -256,7 +256,7 @@ With all these components in place - now it's time to trigger pipeline via webho
 
     ![updated-Nordmart-apps-GitOps-config](images/updated-nordmart-apps-gitops-config.png)
 
-  > For pushes to `main` branch, your application is automagically updated in the `<TENANT_NAME>-dev` namespace.
+  > For pushes to `main` branch, your application is automatically updated in the `<TENANT_NAME>-dev` namespace.
```

Comment:
;(

Reply:
either update vocabulary or fix it for now, for now I just use accepted word